ethash/src/compute.rs

```diff
@@ -152,9 +152,10 @@ pub fn quick_get_difficulty(header_hash: &H256, nonce: u64, mix_hash: &H256, pro
 
 			let buf = buf.assume_init();
 
+			const HASH_BYTES_LENGTH: usize = 32;
```

Comment:
I'm bad at naming, so any other suggestions are welcome :)

Reply:
It'll do

Reply:
Maybe `KECCAK_LEN`?

Reply:
No, because `KECCAK_LEN` would be the total length right?